# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../blitz/gcc43.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Fix compiling issue with gcc >= gcc 4.3
diff -ruN blitz-0.6-orig/blitz/funcs.h blitz-0.6/blitz/funcs.h
--- blitz-0.6-orig/blitz/funcs.h	2002-07-02 21:01:07.000000000 +0200
+++ blitz-0.6/blitz/funcs.h	2010-08-25 09:58:16.552003947 +0200
@@ -44,6 +44,8 @@
  #include <blitz/prettyprint.h>
 #endif
 
+#include <cstdlib>
+
 BZ_NAMESPACE(blitz)
     
 /* Helper functions */
diff -ruN blitz-0.6-orig/blitz/prettyprint.h blitz-0.6/blitz/prettyprint.h
--- blitz-0.6-orig/blitz/prettyprint.h	2001-01-24 21:22:50.000000000 +0100
+++ blitz-0.6/blitz/prettyprint.h	2010-08-25 09:59:10.104007016 +0200
@@ -38,6 +38,8 @@
 #ifndef BZ_PRETTYPRINT_H
 #define BZ_PRETTYPRINT_H
 
+#include <cstdlib>
+
 BZ_NAMESPACE(blitz)
 
 class prettyPrintFormat {
